server/public/model/utils.go

```diff
@@ -487,39 +486,15 @@ func ArrayToJSON(objmap []string) string {
 	return string(b)
 }
 
-// Deprecated: ArrayFromJSON is deprecated,
-// use SortedArrayFromJSON or NonSortedArrayFromJSON instead
 func ArrayFromJSON(data io.Reader) []string {
 	var objmap []string
+
 	json.NewDecoder(data).Decode(&objmap)
 	if objmap == nil {
 		return make([]string, 0)
 	}
```

Comment:
Should we add the duplicate check that was added to the new functions. As I recall @agnivade was pretty insistent on this check.

Reply:
Yeah, it'd be good to keep that. It's an orthogonal problem, but sometimes the DB doesn't have strong integrity checks and if the input is not checked properly, duplicate elements might get inserted.

Reply:
Done.